metashare/api/sf_run_flow.py

```diff
@@ -270,7 +270,7 @@ def run_flow(*, cci, org_config, flow_name, project_path):
 def delete_org(scratch_org):
     """Delete a scratch org by deleting its ActiveScratchOrg record
     in the Dev Hub org."""
-    devhub_username = scratch_org.owner.sf_username
+    devhub_username = scratch_org.owner_sf_id
```

Comment:
@wlonk I think this change was required to actually make it delete with the creating username (not the owner's current `sf_username`).

Reply:
I think I may have had a `.`/`_` visual confusion moment there, but this looks right.